contracts/modules/common/BaseTransfer.sol

```diff
@@ -43,9 +43,18 @@ abstract contract BaseTransfer is BaseModule {
         bytes data
     );
     // *************** Internal Functions ********************* //
+    /**
+    * @notice Make sure a contract call is not trying to call a module, the wallet itself, or a supported ERC20.
+    * @param _wallet The target wallet.
+    * @param _contract The address of the contract.
+     */
+    modifier authorisedContractCall(address _wallet, address _contract) {
```

Comment:
Maybe prefix the modifier with `only` to be consistent with our other modifiers?